src_ext/dune-uutf-src

```diff
@@ -0,0 +1,6 @@
+(library
+  (name uutf)
+  (public_name uutf)
+  (flags :standard -w -3)
+  (libraries cmdliner)
```

Comment:
This pulls in a dependency on `cmdliner` for `opam-core` which breaks the 4.03+ requirement. It's OK just to drop it - `cmdliner` is used for the `utftrip` program, which isn't being built